addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -279,7 +279,15 @@
     }
     @for $level from 1 through 3 {
         .o_we_sublevel_#{$level} {
+            --o-we-checkbox-hover-color: #{$o-we-fg-light};
+            --o-we-sub-option-field-spacing: #{$o-we-sidebar-content-field-spacing * .5};
+
             @extend %o_we_sublevel;
+            color: $o-we-fg-light;
+
+            & + .o_we_sublevel_#{$level + 1} {
+                padding-bottom: 4px;
```

Comment:
I suppose it comes from the suggestion of SRI, but it looks like it's not the expected result. It adds space in a place where it doesn't make sense, or maybe I'm missing something.
See here:
![image](https://github.com/user-attachments/assets/9efdb3c7-f747-4a4a-af0b-dc02344e52cb)

(Anyway, add a comment here to explain the purpose of this code, as it's not obvious.)